index.js

```diff
@@ -339,6 +344,31 @@ function addTest(testTitle, testCaseId, buildUrl) {
   }
 }
 
+function getAttachment(attachmentFolder, testTitle) {
+  const attachmentsFolder = path.join(process.cwd(), `${attachmentFolder}`)
+
+  if (fs.existsSync(attachmentsFolder)) {
+    const attachmentPath = getFiles(attachmentsFolder).find(filename => filename.includes(testTitle))
```

Comment:
can you please mention somewhere (in doc/readme) how are you attaching screenshots, please?
It is not obvious at all

Reply:
Done